ubi9-openjdk-17-runtime.yaml

```diff
@@ -0,0 +1,51 @@
+# This is an Image descriptor for Cekit
+
+schema_version: 1
+
+from: "ubi9-minimal"
+name: &name "ubi9/openjdk-17-runtime"
+version: &version "1.13"
+description: "Image for Red Hat OpenShift providing OpenJDK 17 runtime"
```

Comment:
Shouldn't this have a more generic explanation? Like, for example, `UBI 9 OpenJDK 17 runtime image`

Reply:
Yes. Let me double-check where that field ends up.